tests/unit/models/marker-test.js

```diff
@@ -29,22 +29,22 @@ test('a marker can truncated to an offset', (assert) => {
 
 test('a marker can have a markup applied to it', (assert) => {
   const m1 = new Marker('hi there!');
-  m1.addMarkup(new Markup('b'));
+  m1.addMarkup(Markup.create('b'));
```

Comment:
Unsure these tests should exist like this- perhaps they should go though the builder?